packages/catalog-server/package.json

```diff
@@ -64,13 +65,21 @@
       "output": []
     },
     "emulators:start": {
-      "command": "firebase emulators:start --project wc-catalog",
+      "command": "firebase emulators:start --project wc-catalog --import firebase-data",
       "service": true,
       "files": [
```

Comment:
Was wondering the exported file should be in `files`, but I think not because the only effect of that would be to restart it when you export, which doesn't seem useful. (Not actionable, just noting for the record).

Reply:
That was my thought too. I didn't see how Wireit considering them input or output was useful.